lib/pages/StudentDriverDetailsPage.dart

```diff
@@ -86,10 +88,15 @@ class _StudentDriverDetailsPageState extends State<StudentDriverDetailsPage> {
         onPressed: () async {
           final drivenTimeDao =
               (await StudentDriversDatabase.instance).drivenTimeDao;
-
           var drivenTime = DrivenTime(
               null, args.id, DateTime.now().add(Duration(minutes: 90)), 90);
+
           await drivenTimeDao.insertTime(drivenTime);
+          if(totalTimeSoFar + 90 >= 2700) {
```

Comment:
Co tu się dzieje ?

Reply:
chwilowe sprawdzanie czy już jest więcej niż 30h, oczywiście docelowo powinno być to zrobione tak żeby sprawdzać czas jaki użytkownik doda

Reply:
też powinno być tam 1800 min a nie 2700

Reply:
Wrzuciłbym tą liczbę do zmiennej

Reply:
to i tak zostanie usunięte jak zrobi się zadanie z dodawaniem godzin